lightning-invoice/src/utils.rs

```diff
@@ -931,10 +940,11 @@ mod test {
 		];
 
 		let description_hash = crate::Sha256(Hash::hash("Description hash phantom invoice".as_bytes()));
-		let invoice = ::utils::create_phantom_invoice_with_description_hash::<EnforcingSigner,&test_utils::TestKeysInterface>(Some(payment_amt), None, 3600, description_hash, route_hints, &nodes[1].keys_manager, Currency::BitcoinTestnet).unwrap();
+		let invoice = ::utils::create_phantom_invoice_with_description_hash::<EnforcingSigner,&test_utils::TestKeysInterface>(Some(payment_amt), None, 4200, description_hash, route_hints, &nodes[1].keys_manager, Currency::BitcoinTestnet).unwrap();
```

Comment:
tiny nit: missing whitespace before `&test_utils`.

Also, is there any particular reason why the expiry is set to `4200` here and `3600` everywhere else? If so, maybe use a variable to clarify the difference in the variable name?

Reply:
Thanks!
So I just made it different to the "default" expiry to make it pretty clear it is set. (But seems pretty useless since the expiry is definitely required now).

So I can make it 3600 again or make a variable. Don't think it matters anymore.